projects/1.21/assets/ifr25ku/miragefairy2024/lang/zh_cn.json

```
@@ -0,0 +1,1094 @@
+{
+  "advancements.miragefairy2024.aura_reflector_furnace.description": "建造一个灵气反应熔炉，并向其中添加灵魂燃料。",
+  "advancements.miragefairy2024.aura_reflector_furnace.title": "自我聚合",
+  "advancements.miragefairy2024.bag.description": "制作一个能存放一定数量物品的袋子",
+  "advancements.miragefairy2024.bag.title": "探索时间到！",
+  "advancements.miragefairy2024.calculite.description": "从夏尔皮耶·卢米娜拉的特殊掉落物中获得理天石",
+  "advancements.miragefairy2024.calculite.title": "混沌边缘",
+  "advancements.miragefairy2024.chaos_cube.description": "击败出现在滴水石洞穴遗迹中的混沌立方体",
+  "advancements.miragefairy2024.chaos_cube.title": "魔法控制程序",
+  "advancements.miragefairy2024.chaos_stone.description": "从风化的古代遗迹和世界各地的其他地点获取混沌石",
+  "advancements.miragefairy2024.chaos_stone.title": "科学的未知世界",
+  "advancements.miragefairy2024.deep_fairy_forest.description": "游历这片土地，追寻深处的妖精树海（幽暗仙女之森）。",
```

```suggestion
  "advancements.miragefairy2024.deep_fairy_forest.description": "游历主世界并发现幽暗仙女之森",
```
`地上`从口语上指主世界
至于地名的话，英语和日语都是群系术语本体，所以这边也用术语本身